### ISSUE TITLE
Update website images and glossary

### ISSUE THREAD

**Issue by blairlyons** [simularium/simularium-website#310]
## What needs to happen?
The image on the landing page and the visual glossary image need to be updated, as well as some of the copy accompanying the visual glossary.

Landing page image should be updated to the version from the Nat Methods manuscript. (@thao-do has the final version)

Visual glossary needs an updated screen capture including the new camera buttons. 
Here's a capture: <img width="1792" alt="Screen Shot 2022-02-01 at 2 26 04 PM" src="https://user-images.githubusercontent.com/12102071/152061987-605bde76-96fb-4f60-bf32-42a65cbe3133.png">
(@thao-do can you update the glossary image with the overlays?)

Visual glossary also needs updated copy:

1. LOAD MODEL MENU
Use this menu to choose an example model to view, or to select a file from a public cloud URL or from your local computer. You can also load a file locally from your computer by dragging it into the viewport.

7. CAMERA CONTROLS, from top to bottom:
- ROTATE: When this mode is active, click and drag to rotate the view in 3D. This mode is active by default.
- PAN: When this mode is active, click and drag to pan around the view.
- FOCUS: When this mode is active, the selected agent becomes the camera target and gets centered in the view. This mode is active by default.
- ZOOM IN: Move the camera in towards the camera’s current target (either the camera look at position by default, or a followed object).
- ZOOM OUT: Move the camera away from the camera’s target.
- RESET CAMERA: Reset the 3D camera to the default (original) position and orientation.

## Why should we do this?
To keep the website up to date, especially since we anticipate a publication soon.

## When does this need to get done?
Feb 2022

**Comment by schoinh** [comment]
I've already started working on this so definitely don't worry about it this time, but in the future if you want just the glossary text updated, feel free to submit a PR to update this file! Don't even have to leave GitHub :) This file feeds the React UI.  

https://github.com/simularium/simularium-website/blob/main/src/constants/visual-glossary.ts

I will also make it so that the glossary and the figure display side by side on larger screens, as a part of this ticket.

**Comment by thao-do** [comment]
@blairlyons This is the figure in NatMeth, if we using it as-is, then maybe we'd like to update the caption to have "a", "b", & "c" too? Or should I break it up to accompany the current text we have? 


![Simularium_overview_web](https://user-images.githubusercontent.com/25212364/152452172-ec62d95d-61c4-4e0c-bc8c-4d9c9a6f8616.png)

**Comment by blairlyons** [comment]
@thao-do Sorry I forgot that part, we only have part B on the website right now, and we want to add part C. So the new figure will be just updated part B without the "b. Data flow" and then part C below it with "Agent representation options" as the subtitle

**Comment by thao-do** [comment]
Here is the landing page figure: 

>![Simularium_overview_web_4](https://user-images.githubusercontent.com/25212364/152816777-5e565d05-2b20-4fbd-8323-24a8db741df2.png)

To separate the two parts I added a 50% opacity white boxed to each part; otherwise it'd be a little hard to follow. I also adjusted the font size a bit to adapt for the web. Please let me know if that's not okay. 
I imagine it to look like this when added to the page: 
![Simularium_overview_web_4_expected](https://user-images.githubusercontent.com/25212364/152816862-1f90b906-c12a-4982-ba0c-2147c60209c0.png)

---
And here is the glossary image: 
![Glossary_SimulariumViewer_3](https://user-images.githubusercontent.com/25212364/152817340-1df4e978-9c31-40bc-909f-8fb1109a3ab6.png)
Since we make 2. about the camera controls, the current text needs an update to remove 7. and bump the rest up one number (8.-10. to become 7.-9.).

**Comment by blairlyons** [comment]
@thao-do lgtm!

**Comment by blairlyons** [comment]
Sorry about the screenshot aspect ratio @thao-do, here's a new one.

<img width="1291" alt="Screen Shot 2022-02-07 at 2 52 37 PM" src="https://user-images.githubusercontent.com/12102071/152885775-51c53a62-3cee-427f-bde5-056815a12202.png">

**Comment by blairlyons** [comment]
Sorry about the screenshot aspect ratio @thao-do, here's a new one.

<img width="1585" alt="Screen Shot 2022-02-07 at 3 34 09 PM" src="https://user-images.githubusercontent.com/12102071/152890073-97e175d8-9e7b-4270-94b4-14744672e51d.png">

**Comment by blairlyons** [comment]
sorry about the screenshot aspect ratio @thao-do, here's a new one
<img width="1587" alt="Screen Shot 2022-02-07 at 3 36 45 PM" src="https://user-images.githubusercontent.com/12102071/152890249-b023a964-bace-4203-8962-280bfdc2d8d7.png">

**Comment by blairlyons** [comment]
and @thao-do moving camera controls up to 2 is fine, but we need to keep a number for "CLICK TO SELECT & FOLLOW ANY OBJECT"

**Comment by thao-do** [comment]
I see, I was wondering about that and thought it was included in the camera controls. I guess we should keep the "Click to select & follow any object" as 2, then camera controls is 3 and adjust the rest.

**Comment by thao-do** [comment]
Update image!
![Glossary_SimulariumViewer_5](https://user-images.githubusercontent.com/25212364/152894016-082ebda7-111b-42a6-85c7-226f0ebe8dfe.png)

**Comment by blairlyons** [comment]
@thao-do this one looks good, hopefully I didn't forget anything else 🙃 sorry again!
